src/cameras/list/items/item.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { Checkbox, ListItem, ListItemIcon, ListItemText } from '@material-ui/core'
+
+export const Item = ({ item }) => <ListItem>
+  <ListItemIcon>
+    <Checkbox color={'primary'}
```

Comment:
Let's extract these props into a separate hook so that our component can be a one liner.